applications/openshift/etcd/etcd_auto_tls/bash/bash.sh

```diff
@@ -0,0 +1,7 @@
+# platform = multi_platform_rhel, multi_platform_ocp
```

Comment:
@redhatrises also, name for bash (since no shared)?

Reply:
@shawndwells call them `shared.sh`